assets/xml/objects/object_hana.xml

```diff
@@ -1,13 +1,19 @@
 ﻿<Root>
+    <!-- Assets for the orange graveyard flowers-->
     <File Name="object_hana" Segment="6">
-        <DList Name="object_hana_DL_000000" Offset="0x0" />
-        <DList Name="object_hana_DL_0000A0" Offset="0xA0" />
-        <DList Name="object_hana_DL_000140" Offset="0x140" />
-        <DList Name="object_hana_DL_0001E0" Offset="0x1E0" />
-        <DList Name="object_hana_DL_000280" Offset="0x280" />
-        <DList Name="object_hana_DL_000320" Offset="0x320" />
-        <DList Name="object_hana_DL_0003C0" Offset="0x3C0" />
-        <DList Name="object_hana_DL_000460" Offset="0x460" />
-        <DList Name="object_hana_DL_000500" Offset="0x500" /> <!-- Original name is "flower1_model" -->
+        <!-- Each of these is a single flower out of the 3 -->
+        <DList Name="gGraveyardFlower1DL" Offset="0x0" />
+        <DList Name="gGraveyardFlower2DL" Offset="0xA0" />
+        <DList Name="gGraveyardFlower3DL" Offset="0x140" />
+
+        <!-- Each of these is a single leaf out of the 5 -->
+        <DList Name="gGraveyardFlowerLeaf1DL" Offset="0x1E0" />
+        <DList Name="gGraveyardFlowerLeaf2DL" Offset="0x280" />
+        <DList Name="gGraveyardFlowerLeaf3DL" Offset="0x320" />
+        <DList Name="gGraveyardFlowerLeaf4DL" Offset="0x3C0" />
+        <DList Name="gGraveyardFlowerLeaf5DL" Offset="0x460" />
+
+        <!-- This DL combines all the other DLs to make the whole object -->
+        <DList Name="gGraveyardFlowerModelDL" Offset="0x500" /> <!-- Original name is "flower1_model" -->
```

Comment:
I'm not sure about naming it `FlowerModel`. Maybe just `Flowers`?

Reply:
maybe yea, i named it that cus of the comment and cus i wanted to at least keep the `gGraveyardFlower` part consistent but i mean it does stay kinda consistent with `Flowers` too so yea